<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>robot_arm</class>
 <widget class="QMainWindow" name="robot_arm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>robot_arm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Generate">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Arm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox1">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>50</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>85</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>120</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="link1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>link1</string>
    </property>
   </widget>
   <widget class="QLabel" name="link2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>85</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>link2</string>
    </property>
   </widget>
   <widget class="QLabel" name="link3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>link3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>320</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>joint1</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>280</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>joint3</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>240</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>joint2</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider1">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>200</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text1">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text3">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="slider3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>280</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
